### ISSUE TITLE
TransactWriteItems works or not?

### ISSUE THREAD

**Issue by cyberfox1** [boto/botocore#1726]
AttributeError: 'dynamodb.ServiceResource' object has no attribute 'transact_write_items'

boto3 version is: 1.9.145

**Comment by joguSD** [comment]
Could you run the following:
```
>>> import botocore
>>> import boto3
>>> botocore.__version__
'x.y.z'
>>> boto3.__version__
'x.y.z'
```
Sometimes the version of botocore and boto3 can get out of sync, and even though you have the latest boto3, you might not have the latest botocore.

Might also be worth checking that you don't have any custom models in `~/.aws/models` for dynamodb, as that might leave you with a different/older version than is actually in the version of botocore that you're using.

**Comment by cyberfox1** [comment]
>>> boto3.__version__
'1.9.130'
>>> botocore.__version__
'1.12.130'

**Comment by cyberfox1** [comment]
@joguSD any updates on this?

**Comment by joguSD** [comment]
Ah, looking at this again it seems like you're going through the resource interface and not the client. This operation isn't available on the resource class yet and you'll have to use a boto3/botocore client directly to get this operation.

**Comment by cyberfox1** [comment]
Are there any examples of using the client instead of the resource in Python?

**Comment by eduardovra** [comment]
Do you intend to implement this feature in the resource interface?

**Comment by gadekarabhijit** [comment]
I am also facing the same issue. I am using client and below versions, I am using.
boto3 version:1.9.42
botocore version:1.12.42

**Comment by stealthycoin** [comment]
Its was released in botocore version`1.12.53` so `1.12.42` does not have it. There are no plans currently for adding it to the resource interface, if thats something that would be of interest you should open a separate issue so it can be tracked as a feature request.

Resolving since this is likely fixed, we hadn't added the bot that handles tags to this repo yet. Please repoen if the above doesn't resolve the issue for you.
